### ISSUE TITLE
BUG hemisphere argument no longer plots unilateral structures

### ISSUE THREAD

**Issue by janfpl** [brainglobe/brainrender#152]
Using scene.add_brain_region() method, the hemisphere = 'right' argument no longer plots only one side of a structure - instead both sides plot. Using brainrender 2.0.2.2 works as expected. Tested running the .py file from the terminal in Ubuntu 18.04.

**Comment by FedeClaudi** [comment]
Thanks for letting me know. Can you try upgrading to the latest version please (v `2.0.2.4`)? It should work now

**Comment by janfpl** [comment]
Upgrading to 2.0.2.4 fixed the issue.

**Comment by adamltyson** [comment]
Hi @FedeClaudi, I just tried this on `2.0.2.6` and it works, but it displays the region on the wrong side:
```python
from brainrender import Scene

cam =     {
     'pos': (3341, -31677, -8608),
     'viewup': (-1, 0, 0),
     'clippingRange': (24671, 48886),
   }

scene = Scene()
VISp = scene.add_brain_region("VISp", hemisphere="right")
scene.render(camera=cam, interactive=False)
scene.screenshot(name="hemisphere")
```

![hemisphere](https://user-images.githubusercontent.com/13147259/111613177-2648fa00-87d6-11eb-8bb8-1f1fe7db4415.png)

**Comment by FedeClaudi** [comment]
Thanks, fixed in `v2.0.2.8`
